checks/limits.py

```diff
@@ -0,0 +1,22 @@
+from lib.base import CheckBase
+
+
+class CheckLimits(CheckBase):
+    whitelist = ['DeploymentConfig']
```

Comment:
This check should also work for the following Kinds which implements `PodTemplateSpec` at `spec.template`:

DaemonSet
Deployment
Job
ReplicaSet
ReplicationController
StatefulSet

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#podtemplatespec-v1-core

Reply:
done! I added a base_kinds definition in the CheckBase class so it can be referenced from other tests:
https://github.com/jmelis/manifest-bouncer/blob/337d0969026c4ec3d5d63b1fb9e350a290edcfc0/lib/base.py#L24